Dockerfile

```diff
@@ -53,4 +54,4 @@ RUN curl -Lo nighthawk_output_transform-ubuntu-amd64-v1.0.1.tar.gz https://githu
 RUN mkdir -p /home/appuser/.meshery/config; chown -R appuser /home/appuser/
 USER appuser
 WORKDIR /app/cmd
-CMD ./meshery
+ENTRYPOINT ./meshery
```

Comment:
@bariqhibat why the change from CMD to ENTRYPOINT?

Reply:
no particular reason really, i just followed some of the solution that i found online. should i change it back?

Reply:
Yes, please.

Reply:
sure, just changed it!